Camera.MAUI/Camera.MAUI.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net7.0;net7.0-android;net7.0-ios;net7.0-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net7.0-windows10.0.19041.0</TargetFrameworks>
+		<TargetFrameworks>net8.0;net8.0-android;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net8.0-windows10.0.19041.0</TargetFrameworks>
 		<!-- Uncomment to also build the tizen app. You will need to install tizen by following this: https://github.com/Samsung/Tizen.NET -->
-		<!-- <TargetFrameworks>$(TargetFrameworks);net7.0-tizen</TargetFrameworks> -->
+		<!-- <TargetFrameworks>$(TargetFrameworks);net8.0-tizen</TargetFrameworks> -->
```

Comment:
You're removing support for .NET7 here, but I don't actually think that is a good idea right now, because .NET7 will still be supported by MS for a while, see https://dotnet.microsoft.com/en-us/platform/support/policy/maui.

IMHO you should better add the net8.0 targets in addition to the net7.0 targets, instead of replacing them.

Reply:
That is fine, you have the ability to change that?

Reply:
Well, you can change it yourself. 😉 
Just add a new commit to your branch which fixes it ...

Reply:
Ok, you reverted to net7.0 now, which is not exactly what I suggested, but it's acceptable (since using net8.0 targets is probably not necessary for fixing any of the bugs you quote, right?).

However, you have now effectively removed the net7.0 target, unfortunately ...